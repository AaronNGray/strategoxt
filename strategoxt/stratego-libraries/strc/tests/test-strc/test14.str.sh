#!/bin/sh

java -jar ${STRJ} -i test14.str -la stratego-lib || exit 1 

rm -f test14.str.actual

javac -cp ${STRJ} test14.java || exit 1

java -cp .:${STRJ} -Xmx1024m test14 2>&1 | tail -n 2 > test14.str.actual

diff test14.str.expected test14.str.actual || (echo "Result not expected; test failed"; exit 1)
