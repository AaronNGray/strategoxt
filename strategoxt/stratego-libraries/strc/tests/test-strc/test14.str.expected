	report_with_failure_0_1
	main_0_0
